src/algebra/big_operators/order.lean

```diff
@@ -536,6 +536,21 @@ section canonically_ordered_comm_semiring
 
 variables [canonically_ordered_comm_semiring R] {f g h : ι → R} {s : finset ι} {i : ι}
 
+@[simp]
+lemma _root_.canonically_ordered_comm_semiring.multiset_prod_pos [nontrivial R] {m : multiset R} :
+  0 < m.prod ↔ (∀ x ∈ m, (0 : R) < x) :=
```

Comment:
```suggestion
  0 < m.prod ↔ ∀ x ∈ m, (0 : R) < x :=
```

Reply:
Whoops, that's vestigial from when I had it stated backwards. I don't think this is worth restarting the batch for though.

Reply:
fwiw it's the only PR in its batch